web_development_101/the_basics/how_this_course_will_work.md

```diff
@@ -1,29 +1,29 @@
 ### Welcome to The Odin Project
 
-The Odin Project is an open source community dedicated to providing the best sources information to take you from zero to full-stack developer.  More information at [The Odin Project](https://www.theodinproject.com/about)
+The Odin Project is an open source community project dedicated to providing the best information sources to take you from zero to full-stack developer.  More information can be found at The Odin Project's [about page](https://www.theodinproject.com/about).
 
 ### Introduction
 
-In this unit, we'll learn about how the Web works and start thinking about the basics of computer and web programming.
+In this unit, we'll learn about how the web works and start thinking about the basics of computer and web programming.
 
 Each of the following sections and lessons represents essential baseline knowledge.  Even if you have no intention of becoming a web developer yourself, this material should help you gain a useful understanding of the moving parts involved in creating and serving content on the web.  
 
-We will start by familiarizing you with the internet and your own computer.  You will then learn the basics of front end technologies like HTML, CSS, and Javascript before stepping into the back end with a brief foray into Ruby and Ruby on Rails.  We will finish by learning about databases and topics like Git workflows and software in the cloud.
+We will start by getting familiar with the internet and your own computer.  Next, we'll work on setting up a development environment and learning about Git and GitHub. Then, we'll go over the basics of front end technologies like HTML, CSS, and Javascript before stepping into the back end with a brief foray into Ruby and Ruby on Rails.  We will finish by learning about pair programming so that you can team up to code with others around the world.
 
-By the end of this unit, you should not only understand how the web works, but be able to identify and differentiate each of the technologies that you will be learning in order to build web applications of your own.  You will be able to build a simple webpage, style it, and add minor elements of interactivity while working comfortably from the command line.
+By the end of this unit, you should not only understand how the web works but also be able to identify and differentiate between each of the technologies that you will be learning in order to build your own web applications.  You will be able to build a simple webpage, style it, and add minor elements of interactivity while working comfortably from the command line.
```

Comment:
```suggestion
By the end of this unit, you should not only understand how the web works, but also be able to identify, and differentiate between, all of the technologies that you will use to build your own web applications.  You will be able to build a simple webpage, style it, and add minor elements of interactivity while working comfortably from the command line.
```

Reply:
I made the changes for "each" to "all" and "be learning in order" to "use". However, the comma between the "not only... but also" phrase is incorrect, so I omitted it. (For reference, see Chicago Manual of Style 6.41.) I've also omitted the commas used to offset the phrase "and differentiate between" as this phrase doesn't require setting off and because I feel that it flows better as part of the main sentence structure.